tests/unit/s2n_connection_serialize_test.c

```diff
@@ -592,6 +592,70 @@ int main(int argc, char **argv)
         }
     };
 
+    /* Self-talk: A deserialized session can be serialized */
+    for (size_t i = 0; i < s2n_array_len(config_array); i++) {
```

Comment:
Could we just serialize/deserialize in a loop instead of manually writing out the process twice? Then we'll know we're doing the same thing every time and nothing is different about the second (or third, or fourth) time.